README.md

```diff
@@ -1,2 +1,58 @@
-# indevelopme-coaching-publications
-These are the publications that players on our team created for publications.
+# indevelopme-coaching-publications  
```

Comment:
Is this the actual title? If so, doesn't need to be hyphenated and should be capitalized